pkg/kubelet/apis/config/scheme/testdata/KubeletConfiguration/after/v1beta1.yaml

```diff
@@ -85,5 +85,6 @@ streamingConnectionIdleTimeout: 4h0m0s
 syncFrequency: 1m0s
 topologyManagerPolicy: none
 topologyManagerScope: container
+tracing: {}
```

Comment:
this should be nil / omitted by default after round-trip

Reply:
removed